stdlib/public/core/ReflectionMirror.swift

```diff
@@ -0,0 +1,172 @@
+@_silgen_name("swift_reflectionMirror_displayStyle")
+internal func _getDisplayStyle<T>(_: T) -> CChar
+
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+internal func getChild<T>(of value: T, type: Any.Type, index: Int) -> (label: String?, value: Any) {
+  var nameC: UnsafePointer<CChar>? = nil
+  var freeFunc: NameFreeFunc? = nil
+  
+  let value = _getChild(of: value, type: type, index: index, outName: &nameC, outFreeFunc: &freeFunc)
+  
+  let name = nameC.flatMap({ String(validatingUTF8: $0) })
+  freeFunc?(nameC)
+  return (name, value)
+}
+
+#if _runtime(_ObjC)
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+@_silgen_name("swift_reflectionMirror_quickLookObject")
+internal func _getQuickLookObject<T>(_: T) -> AnyObject?
+
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+@_silgen_name("_swift_stdlib_NSObject_isKindOfClass")
+internal func _isImpl(_ object: AnyObject, kindOf: AnyObject) -> Bool
+
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+internal func _is(_ object: AnyObject, kindOf `class`: String) -> Bool {
+  return _isImpl(object, kindOf: `class` as AnyObject)
+}
+
+@_inlineable // FIXME(sil-serialize-all)
+@_versioned // FIXME(sil-serialize-all)
+internal func _getClassPlaygroundQuickLook(
+  _ object: AnyObject
+) -> PlaygroundQuickLook? {
+  if _is(object, kindOf: "NSNumber") {
+    let number: _NSNumber = unsafeBitCast(object, to: _NSNumber.self)
+    switch UInt8(number.objCType[0]) {
+    case UInt8(ascii: "d"):
+      return .double(number.doubleValue)
+    case UInt8(ascii: "f"):
+      return .float(number.floatValue)
+    case UInt8(ascii: "Q"):
+      return .uInt(number.unsignedLongLongValue)
+    default:
+      return .int(number.longLongValue)
+    }
+  } else if _is(object, kindOf: "NSAttributedString") {
+    return .attributedString(object)
+  } else if _is(object, kindOf: "NSImage") ||
+            _is(object, kindOf: "UIImage") ||
+            _is(object, kindOf: "NSImageView") ||
+            _is(object, kindOf: "UIImageView") ||
+            _is(object, kindOf: "CIImage") ||
+            _is(object, kindOf: "NSBitmapImageRep") {
+    return .image(object)
+  } else if _is(object, kindOf: "NSColor") ||
+            _is(object, kindOf: "UIColor") {
+    return .color(object)
+  } else if _is(object, kindOf: "NSBezierPath") ||
+            _is(object, kindOf: "UIBezierPath") {
+    return .bezierPath(object)
+  } else if _is(object, kindOf: "NSString") {
+    return .text(_forceBridgeFromObjectiveC(object, String.self))
+  }
+
+  return .none
+}
+#endif
+
+extension Mirror {
+  @_inlineable // FIXME(sil-serialize-all)
+  @_versioned
+  internal init(internalReflecting subject: Any,
+  					  subjectType: Any.Type? = nil,
+  					  customAncestor: Mirror? = nil)
+	{
+		let subjectType = subjectType ?? _getNormalizedType(subject, type: type(of: subject))
```

Comment:
Hate to make this my first comment, but... spaces rather than tabs please, because GitHub's rendering mangles the layout.